src/clj/collect_earth_online/db/doi.clj

```diff
@@ -0,0 +1,91 @@
+(ns collect-earth-online.db.doi
+  (:require [clj-http.client :as http]
+            [clojure.java.io :as io]
+            [triangulum.database :refer [call-sql
+                                         sql-primitive]]
+            [triangulum.config  :refer [get-config]]
+            [triangulum.type-conversion :as tc]
+            [collect-earth-online.generators.external-file :refer [create-shape-files]])
+  (:import java.time.format.DateTimeFormatter
+           java.time.LocalDateTime))
+
+(def base-url (:url (get-config :zenodo)))
+
+(defn req-headers
+  []
+  (let [auth-token (:api-key (get-config :zenodo))]
```

Comment:
The structure is likely to undergo some modifications when transitioning to the new Triangulum version, which will involve placing it under the :product-ns namespace.

Reply:
The structure is likely to undergo some modifications when transitioning to the new Triangulum version, which will involve placing it under the :product-ns (ceo) namespace.